### PR TITLE
add private key and cert combination

### PR DESCRIPTION
**Pull request by Sispheor**
Just the rebase of the #14 PR from the current master branch.

### CODE REVIEW

**Comment by jdauphant** [comment]
@Sispheor ok thanks, did you test it ?
Also after, re-reading, I think none of the checks are necessary, the combine should done all the check by default, what do you think ?

**Comment by Sispheor** [comment]
Yes, it works perfectly !

**Comment by Sispheor** [comment]
you mean, when we use stats?

File: tasks/main.yml [hunk 1]
```diff
@@ -84,3 +86,14 @@
...
+  - stat: path={{ ssl_certs_combined_path }}
+    register: stat_combined
+    tags: [ssl-certs,configuration]
+
+  - name: Combine key and cert
+    assemble:
+      dest: "{{ ssl_certs_combined_path }}"
+      src: "{{ ssl_certs_path }}"
+      regexp: "({{ ssl_certs_privkey_path | basename }}|{{ ssl_certs_cert_path | basename }})"
+    when: (not stat_combined.stat.exists) or generate_key_result|changed or generate_cert_result|changed or copy_result_private_key|changed or copy_result_ssl_certificate|changed
```
Comment (jdauphant) [inline tasks/main.yml]: @Sispheor  If we remove all the register added by the PR, this when and the stat task just bellow:
it's should work exactly the same  it's normally the way assemble works)
I will prefer a simplification in that case.
I am 95% sure, it's for that I asking a second point of view

Comment (Sispheor) [reply tasks/main.yml]: OK, I'll test it.

**Comment by Sispheor** [comment]
You were right, I've tested both case:
- the combined file doesn't exist
- private key file of cert file is updated

In both case the assembly module create or update the file accordingly.

I've updated the PR.

**Comment by jdauphant** [comment]
@Sispheor Thanks

**Pull request merged**